ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/Entry/Plugin.cs

```diff
@@ -210,6 +210,7 @@ protected override LoadReturnCode OnLoad(ref string errorMessage)
     {
       Init();
     }
+    // need investigation in seq of specific excpetions thrown (FileNotFound, TypeInitialization)
     catch (Exception ex)
```

Comment:
I think you can probably begin using the IsFatal() method Jedd has created. Though if this is at the topmost level we probably mneed a more specific way to handle IsFatal() to inform the user

Reply:
It probably just needs the !IsFatal() patern then in the case